swebench/harness/modal_eval/run_evaluation_modal.py

```diff
@@ -75,7 +75,7 @@ def _get_sandbox(self, timeout: int | None = None):
             timeout=timeout,
             cpu=4,
             mounts=[
-                modal.Mount.from_local_file(
+                self.image.add_local_file(
```

Comment:
Ah, so the `mounts=` parameter as a whole is deprecated. You can pass `self.image.add_local_file(...)` to `image=` and that should work. I think this won't work right now because this returns an `Image` object, not a `Mount`.

Thanks for fixing!

Reply:
Ah! gotcha, will do that update right now, thank you so much!